hugegraph-store/pom.xml

```diff
@@ -49,6 +49,7 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <log4j2.version>2.15.0</log4j2.version>
+        <top.level.dir>${project.basedir}/..</top.level.dir>
```

Comment:
WARN: ensure we don't have multi `top.level.dir` in project (if we have & need use it, add a comment for it)

Reply:
Are you suggesting that we should only retain the `top.level.dir` property in the root pom? 

Additionally, provide all the occurrences of it:

<img width="764" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/79143929/beed346c-0805-4cab-abf3-d2a7513e105a">

Reply:
THX, just for warning